configure

```diff
@@ -690,7 +690,11 @@ def configure_library(lib, output):
     if default_libpath:
       default_libpath = '-L' + default_libpath
     (pkg_libs, pkg_cflags, pkg_libpath) = pkg_config(lib)
-    cflags = pkg_cflags.split('-I') if pkg_cflags else default_cflags
+    # Remove empty strings from the list of include_dirs
+    if pkg_cflags:
```

Comment:
Minor nitpick. What if `pkg_cflags` is a blank string with more than one whitespace characters? `cflags` would become an empty list. Is that okay?